web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -43,7 +51,19 @@ openerp.web_translate_dialog = function (instance) {
             this.view_type = parent.fields_view.type || '';
             this.$view_form = null;
             this.$sidebar_form = null;
-            this.translatable_fields_keys = _.map(this.view.translatable_fields || [], function(i) { return i.name;});
+            if (typeof(field) !== 'undefined') {
```

Comment:
This line can be rewritten as: `if (!!field) {`, it's an old JavaScript trick, casts to boolean as you are looking for a **truthy** value. :wink: